test/library/packages/UnitTest/StringSpace.good

```diff
@@ -0,0 +1,5 @@
+./UnitTest.chpl:149: error: Strings differ: 'Just a string' != 'Just a string with space'
```

Comment:
This is a little inconsistent - you use English for the other message, but the symbol `!=` here

Reply:
I will update all the symbols and will try to make everything consistent.